Lib/idlelib/browser.py

```diff
@@ -19,14 +19,45 @@
 from idlelib.tree import TreeNode, TreeItem, ScrolledCanvas
 from idlelib.windows import ListedToplevel
 
+
 file_open = None  # Method...Item and Class...Item use this.
 # Normally pyshell.flist.open, but there is no pyshell.flist for htest.
 
+
+def _traverse_node(node, name=None):
+    """Return the immediate children for a node.
+
+    Node is the current node being traversed.  The return value is
+    a tuple with the first value being a dictionary of the
+    Class/Function instances of the node and the second is
+    a list of tuples with (lineno, name).
```

Comment:
I've changed the docstring.  I think he left `name` as a parameter because it was used it in existing code.  Also, the values returned from this function are pretty confusing.  I think he tried to pull it "as-is" out of the existing code and modify it as little as possible, but it's not very clear.